README.md

```diff
@@ -43,6 +43,7 @@ Here you'll find community contributions to the Gotify project.
 - [eikendev/action-gotify](https://github.com/eikendev/action-gotify) A GitHub action to send notifications via `gotify/server`.
 - [Enchufadoo/gotify-fox](https://github.com/Enchufadoo/gotify-fox) A Firefox addon for sending messages to `gotify/server`.
 - [gigigig/bedrock-docker-gotify](https://github.com/gigigig/bedrock-docker-gotify) Sends Player Events from Minecraft Bedrock Docker Server to `gotify/server`.
+- [iGotify-Notification-Assistent](https://github.com/androidseb25/iGotify-Notification-Assistent) - Assistent to send Push Notification to iOS Devices with Websocket connection from `gotify/server`
```

Comment:
Awesome! Looks pretty easy to use. I've one little question about security: The docker image contains a service_account with a private key. Is this secure enough that a malicious third-party cannot abuse this? E.g. send notifications to a device from another user?

Reply:
It might be possible, I don't know for sure, but the malicious person needs the device token from a device to send the notification, but I can also encrypt the file in the container.

Reply:
@jmattheis i updated now the package and encrypted the file

Reply:
Okay, not sure if encryption helps there because the application has to encrypt it to use it. Thus, the decryption secret should be somewhere in the container.

But yeah, if that's secure enough, I've no objections (:.

Reply:
yes it's secure enough for this purpose